src/app/ui/team-page/team-page.component.html

```diff
@@ -1,6 +1,6 @@
-<div class="container" *ngIf="auth.user$ | async as user">
-  <team-members [currentUser]="user"></team-members>
+<div class="container" *ngIf="auth.loginInfo | async as info">
+  <team-members [currentUser]="info.userInfo" [seasonId]="info.seasonId"></team-members>
```

Comment:
you can also use async pipe here via: `[seasonId]"="seasonId$ | async"`
with `this.seasonId$ = this.seasonService.getEnabledSeasonId();` in `team-page.component.ts`